sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -483,6 +483,7 @@ Command *
 Scheduler::GraphBuilder::addCG(std::unique_ptr<detail::CG> CommandGroup,
                                QueueImplPtr Queue) {
   std::vector<Requirement *> Reqs = CommandGroup->getRequirements();
+  std::vector<detail::EventImplPtr> Events = CommandGroup->getEvents();
```

Comment:
Please, move this line closer to where it's used.

Reply:
Moving this line is probably going to cause more problems than it's worth.

The CG gets std::moved when the NewCmd is made.  I would either have to:
1) Not std::move it and deal with it later
2) Change the interface of ExecCGetcetc